docs/wiki/[User-Guide]-Starter-Module-Terraform-Microsoft-Cloud-For-Sovereignty.md

```diff
@@ -0,0 +1,96 @@
+<!-- markdownlint-disable first-line-h1 -->
```

Comment:
Given the discussion in the other chat, can we add a banner up at the top here that states it is `Public Preview` and only tested with the `local` option with `local state`. "Feel free to use the other options at your own risk"?

Also make it super clear why we are asking them to set the bootstrap inputs in the table. 

The `starter` input being `false` is still incorrect though and needs to be updated to the correct name.

Reply:
@VeronicaSea I've updated the docs with what should be all of the ADO, GH, and Local methods. Feel free to remove any that you don't feel we are comfortable with supporting for this release.